sky/resources.py

```diff
@@ -1204,13 +1242,33 @@ def from_yaml_config(
         common_utils.validate_schema(config, schemas.get_resources_schema(),
                                      'Invalid resources YAML: ')
 
+        def _merge_labels(base_resource_config: Dict[str, Any],
+                          override_config: Dict[str, Any]):
+            # Returns the base labels "updated" with the override labels.
+            labels = base_resource_config.get('labels')
+            # Merge the labels from the base and override resources.
+            if labels is not None:
```

Comment:
Does this mean that if there is no global labels specified, the local labels will not be used? Should we do the following suggestion instead?